### ISSUE TITLE
Reading binary files generated by fix_phonon in example

### ISSUE THREAD

**Issue by vishankkumar** [lingtikong/phana#2]
Dear Developer,

Thank you for providing the post processing code to analyze the phonon dispersion curves from lammps data. Recently, I tried the example 4-Graphene that comes with the repository and am facing an issue in reading the binary file. When I launch this command:
 **phana Graphene.bin.6000000 < in.disp**
It launch the 'phana' code but ask for some information about the path for phonon band structure which is supposedly missing. The exact error is:
**Please input the filename to output the dispersion data [pdisp.dat]:
Please input the start q-point in unit of B1->B3, q to exit [0 0 0]: Please input the end q-point in unit of B1->B3: Please input the # of points along the line [6]: 
Please input the start q-point in unit of B1->B3, q to exit [0 0 0]: Please input the end q-point in unit of B1->B3: Please input the # of points along the line [2]: 
Please input the start q-point in unit of B1->B3, q to exit [0.5 0.5 0]: Please input the end q-point in unit of B1->B3: Please input the # of points along the line [2]: 
Segmentation fault (core dumped)**

I have build lammps with USER-Phonon, USER-FFTW3 which is required for fix phonon. Do you think I should check for the installation of phana code? I did not install it separately, I guess while compiling lammps, it also compiled this code. Let me know if I am missing some steps.

Thanks in advance for your response.

Regards,
Vishank

**Comment by vishankkumar** [comment]
Actually, I found out that it is not reading the qpoints from the file 'in.disp' and throwing an error as it does not know the q points for calculation. If I provide it manually, it is working fine. May be it needs some fix in the code to read it from the input file as suggested in the example.

Thanks,
Vishank

**Comment by lingtikong** [comment]
Dear Vishank,

Please try to run phana by:

phana *Graphene.bin.6000000*

*directly and follow the menu step by step to check if it is working.*

*Thanks.*

*Lingti Kong
*

在 2019-10-22 23:56, vishankkumar 写道:

**Comment by vishankkumar** [comment]
Dear Prof. Ling Kong,

Thank you for your suggestion. Actually, in the meantime I tried the same and it works.

Moreover, I looked carefully in the "in.disp" file  of the example repository of lammps. There is an extra line with "1" in place of the file name "pdisp.dat", I corrected it and it worked fine. The correct version is:

100
1
2
pdisp.dat
0   0    0       ! Gamma
0.5 0.5  0       ! M
21
0.5 0.5  0       ! M
0.3333333333 0.6666666666666 0. ! K
11
0.3333333333 0.6666666666666 0. ! K
0.  0   0        ! Gamma
21
q
0

The code works fine, its a mistake in the example repository of lammps I guess.

Thanks for your timely response and providing a useful code to the research community.

Regards,
Vishank
